### ISSUE TITLE
Avatars floating

### ISSUE THREAD

**Issue by broehl** [mozilla/hubs#2324]
We had a meetup of about 20 people, and one of them (wearing a robot avatar) was floating high above the others. Flying was disabled in the room, so it wasn't that. There was nothing below them except the floor we were all standing on, and she remained floating high no matter where in the room she went.

I've seen this happen in other rooms as well.

**Comment by misslivirose** [comment]
@johnshaughnessy - Is it possible if a scene has a waypoint in the air that there is some combination of permissions that would interact with room flying in some way? I'm curious what could cause something like this. 

Would be helpful to get information about the scene(s) that this was happening in, confirm that the user wasn't promoted (which overrides the 'No flying' rule), and to confirm that the avatar was a standard one (sounds like it was).

**Comment by broehl** [comment]
The scene was Morning Dew. The user was not promoted, and her avatar was one of the robots.

**Comment by johnshaughnessy** [comment]
If the user was flying up and down (depending on viewing direction), that sounds like fly-mode is activated. I suspect it is something else, which was suggested today by someone during our standup meeting: that the user could have been using a modified robot where the Head node was mid-configured. We have a series of cute pet avatars featured right now that exhibit this problem. I believe they will be removed and re-added after this is fixed. 

Example avatar: https://hubs.mozilla.com/avatars/aEAz0w1

The camera spawns at "viewing height", but notice that this is aligned with the bottom of the avatar:

![image](https://user-images.githubusercontent.com/4072106/78622670-9915cb00-783a-11ea-9d3a-1a322367d19b.png)

**Comment by broehl** [comment]
Yes, in fact that black-and-white cat avatar looks like the one she was using.

**Comment by midnightinfour** [comment]
Friends and I encountered the same issue on an environment with flying disabled.  User was new and selected one of the objects in the scene using the Objects menu in the top right of in-scene interface, placing User's avatar at whatever default position and plane the chosen Object had defined for itself.

In this case, User ended up at Object's midpoint, which was a plane high above (maybe Y:25) the Scene's floorplan level of Y:0.  User consequently had no constraints on planar movement and was functionally able to "fly" down to the ordinary floorplan.
